spec/rspec/support/recursive_const_methods_spec.rb

```diff
@@ -33,7 +36,6 @@ class Bar < Parent
         end
 
         it 'does not blow up on buggy classes that raise weird errors on `to_str`' do
-          allow(Foo::Bar).to receive(:to_str).and_raise("boom!")
```

Comment:
I'd restore this and just add an implementation that doesn't raise? Seems like a "safer" change

Reply:
👍 Done